pkg/git/git.go

```diff
@@ -26,16 +26,19 @@ import (
 	"golang.org/x/xerrors"
 )
 
-func run(logger *zap.SugaredLogger, args ...string) error {
+func run(logger *zap.SugaredLogger, dir string, args ...string) (string, error) {
 	c := exec.Command("git", args...)
 	var output bytes.Buffer
 	c.Stderr = &output
 	c.Stdout = &output
+	if dir != "" {
+		c.Dir = dir
```

Comment:
nit: I'm a bit unclear why the assignment is wrapped in an `if`.  What is `c.Dir`at this point and what are we protecting against with the `if`?

Reply:
Clarified with a comment.